NOTES.md

```diff
@@ -1,3 +1,132 @@
+# Release 2.1.0-beta (Forbidden Planet)
+
+-- issues up-to-date with the 2.1 milestone as of 2015-07-12 20:00 UTC
```

Comment:
No, those are just for book keeping while things might still be changing. I'll remove them before merging.